packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -206,6 +213,25 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+/**
+ * Returns true if compat test version is below the one provided as minimum version.
+ * It helps to filter out lower verions configs that the ones intended to be tested on a
+ * particular suite.
+ */
+export function isCompatVersionBelowMinVersion(minVersion: string, config: CompatConfig) {
+	let lowerVersion: string | number = config.compatVersion;
+	// For CrossVersion there are 2 versions being tested. Get the lower one.
+	if (config.kind === CompatKind.CrossVersion) {
+		lowerVersion =
+			semver.compare(config.compatVersion as string, config.loadVersion as string) > 0
+				? (config.loadVersion as string)
+				: config.compatVersion;
+	}
+	const compatVersion = getRequestedVersion(testBaseVersion(lowerVersion), lowerVersion);
+	const minReqVersion = getRequestedVersion(testBaseVersion(minVersion), minVersion);
```

Comment:
I remember I looked at this and commented about it. But this will throw if the minVersion is not in correct format right? Just want to make sure that if someone uses an incorrect version, they will become aware and can fix it.

Reply:
Yes, from your comment, I moved the error handling deeper into getRequestedVersion.